Formula/datomic.rb

```diff
@@ -15,6 +15,21 @@ def install
     %w[transactor repl repl-jline rest shell groovysh maven-install].each do |file|
       (bin/"datomic-#{file}").write_env_script libexec/"bin/#{file}", Language::Java.java_home_env
     end
+
+    # create directory for datomic data and logs
+    (var+"lib/datomic").mkpath
+
+    # install free-transactor properties
+    data = var/"lib/datomic"
+    (etc+"datomic").mkpath
+    cp libexec/"config/samples/free-transactor-template.properties", etc/"datomic/free-transactor.properties"
```

Comment:
This can be written as
```ruby 
(etc/"datomic").install libexec/"config/samples/free-transactor-template.properties" => "/free-transactor-template.properties"
```